src/context/spt_js/dg_table.js

```diff
@@ -3064,18 +3064,18 @@ spt.dg_table._search_cbk = function(evt, bvr)
 
     var pat = /TileLayoutWdg|CollectionLayoutWdg/;
     //if (pat.test(class_name)) { }
-    var attr_list = ['expand_mode','show_name_hover','scale','sticky_scale','top_view', 'bottom_view','aspect_ratio','show_drop_shadow', 'title_expr', 'overlay_expr', 'overlay_color', 'allow_drag', 'upload_mode','process','gallery_align','detail_element_names','hide_checkbox'];
+    var attr_list = ['expand_mode','show_name_hover','scale','sticky_scale','top_view', 'bottom_view','aspect_ratio','show_drop_shadow', 'title_expr', 'overlay_expr', 'overlay_color', 'allow_drag', 'upload_mode','process','gallery_align','detail_element_names','hide_checkbox', 'ingest_custom_view'];

```

Comment:
Sure np, I did this because I noticed that line 3066 is commented out, which means that this attribute list is checked for all layout widgets. I like this dynamic way of grabbing attributes since it makes for less lines of code.